packages/valory/connections/p2p_libp2p/consts.py

```diff
@@ -22,6 +22,22 @@
 import platform
 
 
+DEFAULT_LEDGER_LIBP2P_NODE = "cosmos"  # Secp256k1 keys
```

Comment:
Don’t think that’s appropriate here. They’re used in tests and not in the package itself